packages/docs/src/examples/v-command-palette/prop-dialog.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <v-container fluid>
+    <v-row>
+      <v-col cols="12" md="4">
+        <v-card>
+          <v-card-title>Default (Center)</v-card-title>
+          <v-card-text>
+            <v-btn block @click="dialogs.default = true">
+              Open Default
+            </v-btn>
+          </v-card-text>
+        </v-card>
+
+        <v-command-palette
+          v-model="dialogs.default"
+          :items="items"
+          placeholder="Search..."
+          @click:item="handleClick"
+        ></v-command-palette>
+      </v-col>
+
+      <v-col cols="12" md="4">
+        <v-card>
+          <v-card-title>Top Position</v-card-title>
+          <v-card-text>
+            <v-btn block @click="dialogs.top = true">
+              Open at Top
+            </v-btn>
+          </v-card-text>
+        </v-card>
+
+        <v-command-palette
+          v-model="dialogs.top"
+          :items="items"
+          location="top"
```

Comment:
To force VDialog to show up from the top we would need a static anchor. ([demo](https://play.vuetifyjs.com/playgrounds/nsUU8Q))
This is a good feature to implement on the VOverlay level anyway.
For now, I would drop this example and the `location` prop.